### ISSUE TITLE
Leaderboard button redirect

### ISSUE THREAD

**Issue by cytronicoder** [hackclub/sinerider#461]
@cytronicoder we need to include a link to the leaderboard on the loading page- just using www.google.com or some placeholder URL for the leaderboard will do. and then @grymmy will actually connect the leaderboard.

_Originally posted by @alhardwarehyde in https://github.com/hackclub/sinerider/issues/207#issuecomment-1520618553_

I feel like the leaderboard can now be put on the loading page on the top right as a button that says "Check out the leaderboard" or just "Leaderboard." Thoughts?

<img width="2240" alt="Screenshot 2023-05-01 at 11 41 30 AM" src="https://user-images.githubusercontent.com/105997270/235403395-0fd9d7e9-afab-47af-80a5-8814c4084d81.png">

**Comment by camdan-me** [comment]
Agreed, this seems like a good idea. Maybe don't show on the first visit.

**Comment by grymmy** [comment]
I think it makes sense to put the leaderboard link down by the "New puzzles" copy down on the bottom.  I would not overcomplicate this by trying to make it not appear on first visit, highly recommend just putting the static link there.

**Comment by cytronicoder** [comment]
Finally added this bad boy in
<img width="547" alt="Screenshot 2023-05-08 at 11 53 23 PM" src="https://user-images.githubusercontent.com/105997270/236870918-c8feb367-3b2e-4f37-888f-7651d978d838.png">
